source/docs/articles/organizations/change-management.md

```diff
@@ -99,29 +132,43 @@ By default, Change Management is not enabled for some types of Organizations. If
 .tg .tg-e3zv{font-weight:bold}
 </style>
 <table class="tg">
-<tr>
-<th class="tg-e3zv">Role</th>
-<th class="tg-e3zv">Organization Dashboard</th>
-<th class="tg-e3zv">Site Dashboard</th>
-</tr>
-<tr>
-<td class="tg-031e">Owner</td>
-<td class="tg-031e">Functionally the same as an Organization Admin, distinguished only by the user/organization's association with a payment instrument.  </td>
-<td class="tg-031e">Same as Organization Administrator.</td>
-</tr>
-<tr>
-<td class="tg-031e">Organization Administrator</td>
-<td class="tg-031e">Only role that can access this dashboard. All permissions are granted.</td>
-<td class="tg-031e">In Pantheon for Enterprise Organizations, the Site is owned by the Organization, and the Organization Administrator functions as site Owner and can:<br>- Manage Sites Service Level<br>- Delete Sites<br>- Manage Settings<br>If the Site is not owned by the Organization, the Organization Administrator functions as a Team Member.</td>
-</tr>
-<tr>
-<td class="tg-031e">Team Member</td>
-<td class="tg-031e">Role does not exist in organizations without Change Management</td>
-<td class="tg-031e">- Manage Settings<br>- Manage Team<br>- Manage Settings<br>- Deploy to Test/Live</td>
-</tr>
-<tr>
-<td class="tg-031e">Unprivileged*</td>
-<td class="tg-031e"> No Access</td>
-<td class="tg-031e">- Become Team Members for sites they create<br>- Can only access sites through invitation, are considered team members.</td>
-</tr>
+    <tr>
+        <th><strong>Role</strong></th>
+        <th><strong>Permissions</strong></th>
+        <th><strong>Enterprise Organization</strong></th>
+        <th><strong>Partner Organization</strong></th>
+    </tr>
+    <tr>
+        <td>Administrator</td>
+        <td>* Create and delete sites</td>
+        <td><center>X</center></td>
+        <td><center>X</center></td>
+    </tr>
+    <tr>
+        <td>Team Member</td>
+        <td>* Deploy code to Test and Live<br/>* Use the Workflow tool on Test and Live<br/>* Change the PHP versions for all site environments<br/>* Add Team Members and Supporting Organizations</td>
+        <td><center>X</center></td>
+        <td><center>X</center></td>
+    </tr>
+    <tr>
+        <td>Developer</td>
+        <td>* Commit code to Dev and Multidev environments<br/>* Add and Remove add-ons to sites. New Relic cannot be removed once added.</td>
+        <td><center>X</center></td>
+        <td><center>X</center></td>
+    </tr>
+    <tr>
+        <td>User in Charge</td>
+        <td>* This is the user who created the site. This designation cannot be transferred.<br/>* Can delete the site he/she created</td>
+        <td><center>X</center></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>Owner</td>
+        <td>* Functionally is the same as an Organization Administrator</td>
+        <td></td>
+        <td><center>X</center></td>
+    </tr>
 </table>
+
+<br>
+To promote a Developer to the Team Member role for a single site, add the user to the Site Team.
```

Comment:
This is a great tip. May want to consider making it move obvious that it's a tip. 

Completely optional, but better still, talk through some use cases of how how CM works in an org.